action/signedaction.go

```diff
@@ -470,3 +471,43 @@ func SignedMigrateStake(
 	}
 	return selp, nil
 }
+
+func SignedClaimRewardLegacy(
```

Comment:
IMO, functions in this file should be auxiliary functions in test files, since the tx builder and signing is clear already.